src/libraries/System.Threading/tests/InterlockedTests.cs

```diff
@@ -603,6 +603,22 @@ public void InterlockedAddAndRead_Multithreaded_Int64()
             Assert.Equal(ThreadCount * IterationCount * Increment, Interlocked.Read(ref value));
         }
 
+        static Action mb;
+        static delegate* <void> pMb;
```

Comment:
`static` state in tests is not pretty. What is this trying to achieve? 

If you are trying to avoid optimizations from kicking in, you can create a noinlineable method that returns the delegate or function pointer.

Reply:
> you can create a noinlineable method that returns the delegate or function pointer.
yes, that would be better.